crypto/libsodium-fork/src/libsodium/crypto_core/ed25519/ref10/ed25519_ref10_sc.c

```diff
@@ -269,20 +324,24 @@ int lte256_modm_batch(const sc25519 a, const sc25519 b, size_t limbsize) {
 
 /* is a == 0 */
 int iszero256_modm_batch(const sc25519 a) {
-	size_t i;
-	for (i = 0; i < 5; i++)
-		if (a[i])
-			return 0;
-	return 1;
+	sc25519_element_t result = a[0] | 0;
+	result |= a[1] | 0;
+	result |= a[2] | 0;
+	result |= a[3] | 0;
+	result |= a[4] | 0;
```

Comment:
I'm abit confused here about the "| 0". `x | 0 == x`, right ? and while I'm sure the compiler would omit that, I can't find any reason to write it. anything that I missed ?

Reply:
I don't think you are missing anything. thanks for the catch